examples/google-ads-account-analyzer-demo/analyzer.py

```diff
@@ -0,0 +1,224 @@
+#!/usr/bin/env python
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Gets the account infomation of the given MCC and login customer."""
+
+
+import argparse
+import sys
+
+from google.ads.googleads.client import GoogleAdsClient
+from google.ads.googleads.errors import GoogleAdsException
+
+
+def account_hierarchy_module(google_ads_client, customer_id):
+    """Print the account hierarchy for the given login customer ID
```

Comment:
Nit: add period here and after args. 
Aside: I wonder if there's a way to enforce this with a linter or automate with `black`?